### ISSUE TITLE
error using cp

### ISSUE THREAD

**Issue by mrmcc3** [boot-clj/boot#502]
I get the following error using 2.6.0 but not 2.5.5

```
java.lang.IllegalArgumentException: No matching method found: copy
                                ...
              boot.tmpdir/add-blob!                        tmpdir.clj:   87
```

To reproduce

``` clojure
(set-env! :resource-paths #{"test"})

(require '[clojure.java.io :as io])

(deftask copy []
  (with-pre-wrap filest
    (-> fileset
        (cp (io/file "build.boot") (-> fileset input-files first))
        commit!)))
```

If my understanding is correct running `boot copy` should replace a file in `test` dir with contents of `build.boot`

**Comment by francisluong** [comment]
As of 2.7.1, there seems to be a typo.  The call to `add-blob!` supplies `blob` as the first argument but I do not see what it is referring to.

https://github.com/boot-clj/boot/blob/master/boot/pod/src/boot/tmpdir.clj#L321-L328
```clojure
  (cp [this src-file dest-tmpfile]
    (let [hash (digest/md5 src-file)
          p'   (path dest-tmpfile)
          d'   (dir dest-tmpfile)]
      (assert ((set (map file dirs)) d')
              (format "dest-dir not in dir set (%s)" d'))
      (add-blob! blob src-file hash *hard-link*)
      (assoc this :tree (merge tree {p' (assoc dest-tmpfile :id hash)})))))
```

**Comment by martinklepsch** [comment]
Old bug but still present. Seems nobody is using `boot.tmpdir/cp` 😹
Opened a PR for a fix in #675.
